cloud/grpc.go

```diff
@@ -100,26 +103,72 @@ func (c *Client) StreamInterceptor() grpc.StreamClientInterceptor {
 		ctx = c.withReqID(ctx)
 		ctx, err := c.reAuthIfExpired(ctx)
 		if err != nil {
-			return nil, errors.Wrap(err, "failed refreshing expired token")
+			return nil, errors.Wrapf(err, "failed refreshing expired token {reqID: %s}", getReqID(ctx))
 		}
 		newStreamer, err := streamer(ctx, desc, cc, method, opts...)
 		if err != nil {
 			s, ok := status.FromError(err)
 			if !ok {
-				return newStreamer, err
+				return nil, fmt.Errorf("%s {reqID: %s}", err.Error(), getReqID(ctx))
 			}
-			err = status.Errorf(s.Code(), fmt.Sprintf("%s {reqID: %s}", s.Err(), getReqID(ctx)))
 			if s.Code() == codes.Unauthenticated {
 				ctx, err = c.reAuthCtx(ctx)
 				if err != nil {
-					return newStreamer, err
+					return nil, fmt.Errorf("%s {reqID: %s}", err.Error(), getReqID(ctx))
 				}
 				return streamer(ctx, desc, cc, method, opts...)
 			}
-			return newStreamer, err
+			return nil, status.Errorf(s.Code(), fmt.Sprintf("%s {reqID: %s}",
+				cleanStatusError(err.Error()), getReqID(ctx)))
+		}
+		return newWrappedStream(ctx, newStreamer), nil
+	}
+}
+
+// wrappedStream  wraps around the embedded grpc.ClientStream, and intercepts the RecvMsg and
+// SendMsg method call.
+type wrappedStream struct {
```

Comment:
`wrappedStream` seems too generic, maybe call it `requestIDErrorStreamWrapper` or something else?